### PR TITLE
Incorporated password strength checker into the register function

### PR DESCRIPTION
**Pull request by cenesdeveloper**
Incorporated password strength checker into the register function.

### CODE REVIEW

**Review by zareentk — APPROVED** [review]
LGTM!

File: app/src/auth.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 from flask import Blueprint, render_template, redirect, url_for, request, flash
 from .models import User, db
 from flask_login import login_user
-
+from passwordStrength import check_password_strength
```
Comment (skydonline) [inline app/src/auth.py]: Add a `.` for local file imports

File: app/src/auth.py [hunk 2]
```diff
@@ -51,7 +51,16 @@ def register():
...
         username = request.form.get("username")
+
+        #Check the Strength of the password
         password = request.form.get("password")
+        result = check_password_strength(password)
+        if not result:
+            flash("Password is not strong enough")
+        else:
+            flash("Password is not strong enough. Here are some suggestions: " + ", ".join(result))
+            return redirect(url_for("register"))
```
Comment (skydonline) [inline app/src/auth.py]: If the password is strong enough, result will be None. So `if not result` is the situation where the password is strong enough, and `else` is when the password is not strong enough.

**Review by skydonline — CHANGES_REQUESTED** [review]
Check comments for imports and result for check_password_strength

File: app/src/auth.py (hunk 1 above)
Comment (cenesdeveloper) [reply app/src/auth.py]: Added a . before the file name passwordStrength.

File: app/src/auth.py (hunk 2 above)
Comment (cenesdeveloper) [reply app/src/auth.py]: Changed the message to display "Password is strong enough".

**Review by skydonline — APPROVED** [review]

**Pull request merged**